great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -881,32 +881,24 @@ def _apply_runtime_environment(
                     property_key,
                     property_value,
                 ) in domain_type_directives.directives.items():
-                    # Insure that new directives augment (not eliminate) existing directives.
-                    existing_property_value = getattr(
-                        rule.domain_builder, property_key.value, None
-                    )
-                    property_value = BaseRuleBasedProfiler._get_effective_domain_builder_property_value(
-                        dest_property_value=property_value,
-                        source_property_value=existing_property_value,
-                    )
-                    setattr(rule.domain_builder, property_key.value, property_value)
+                    try:
+                        # Insure that new directives augment (not eliminate) existing directives.
+                        existing_property_value = getattr(
+                            rule.domain_builder, property_key.value
+                        )
+                        property_value = BaseRuleBasedProfiler._get_effective_domain_builder_property_value(
+                            dest_property_value=property_value,
+                            source_property_value=existing_property_value,
+                        )
+                        setattr(rule.domain_builder, property_key.value, property_value)
+                    except AttributeError:
+                        pass
```

Comment:
I don't love unchecked `try/except`. At the very least, I think we'd want a `log.debug`. Thoughts?